client/src/templates/Introduction/components/CertChallenge.js

```diff
@@ -118,10 +144,9 @@ const CertChallenge = ({
       ? verifyCert(certSlug)
       : createFlashMessage(honestyInfoMessage);
   };
-
   return (
     <div className='block'>
-      {(!isCertified || !canViewCert) && (
+      {userLoaded && isSignedIn && !isCertified && !canViewCert && (
```

Comment:
```suggestion
      {userLoaded && isSignedIn && (!isCertified || !canViewCert) && (
```

Reply:
This was silly - I did not read above conversation.

Reply:
No problem Shaun!